classifiers/reference_quality/tiktoken_length_classifier/__init__.py

```diff
@@ -2,20 +2,22 @@
 import tiktoken
 
 INPUT_EXAMPLE = {
-    "text": "The sun is shinng brigt today."
+    "text": "The sun is shining bright today.",
+    "encoding_model": "cl100k_base"
 }
 
 
 class TiktokenLengthClassifierModel(BaseModel):
     text: str
+    encding_model: str = "cl100k_base"
```

Comment:
here is a typo, `encoding_model`

Reply:
also in the rest of the code, `encoding_name` is always used as a variable name. I would suggest to stick with one

Reply:
Makes sense, I changed it to `encoding_model` in the `init`, `refinery` and `common` code.